lib/api/unittest/CPersistenceManagerTest.cc

```diff
@@ -35,9 +35,11 @@ BOOST_AUTO_TEST_SUITE(CPersistenceManagerTest)
 namespace {
 
 void reportPersistComplete(ml::api::CModelSnapshotJsonWriter::SModelSnapshotReport modelSnapshotReport,
+                           std::string& description,
                            std::string& snapshotIdOut,
                            std::size_t& numDocsOut) {
     LOG_DEBUG(<< "Persist complete with description: " << modelSnapshotReport.s_Description);
+    description = modelSnapshotReport.s_Description;
```

Comment:
This could also capture the snapshot timestamp so tests can assert that is as expected too.